app/test_main.py

```diff
@@ -1 +1,51 @@
-# write your code here
+from unittest import mock
+import pytest
+
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "valid_url, internet_connection, can_access",
+    [
+        pytest.param(
+            True, False, "Not accessible",
+            id="if valid google url is failing"
+        ),
+        pytest.param(
+            False, True, "Not accessible",
+            id="if internet connection is failing"
+        ),
+        pytest.param(
+            True, True, "Accessible",
+            id="if valid url and internet connection are both True"
+        )
+    ]
+)
+def test_for_some(
+        valid_url: bool,
+        internet_connection: bool,
+        can_access: bool
```

Comment:
you can make these names a little bit more descriptive. For example, you can name you func test_can_access_google_page and change the parameter names to `is_valid_url, has_internet_connection, expected_result`
But it's ok, only func name is a little pain.

Reply:
Forget to change test function name)